.github/labels.yml

```diff
@@ -138,11 +138,6 @@
   description: "Make this PR count for hacktoberfest"
   color: "ff7518"
 
-# This label can be added to accept PRs as part of Hacktoberfest
-- name: "hacktoberfest-accepted"
-  description: "Make this PR count for hacktoberfest"
-  color: "ff7518"
-
```

Comment:
I would prefer that this label be retained, unless this label is going to be an organization wide label and is stored/set elsewhere.

Reply:
@kotp It is an org-wide label, this just removed the repo-specific version of it. See https://github.com/exercism/ruby-test-runner/blob/d467bc95255d09c55b72035e2fd1efe16e542209/.github/labels.yml#L137 for proof that it will still be in the labels.yml file. And see https://github.com/exercism/org-wide-files/blob/main/global-files%2F.github%2Flabels.yml#L137 for the global source.